dist/converse.js

```diff
@@ -59128,6 +59130,19 @@ _converse_headless_converse_core__WEBPACK_IMPORTED_MODULE_5__["default"].plugins
         this.model.sendFiles(evt.target.files);
       },
 
+      onDragOver(evt) {
+        evt.preventDefault();
+      },
+
+      onDrop(evt) {
+        /* There are no files to be dropped, this isn’t a file transfer
+         * operation.
+         */
```

Comment:
This comment confuses me. Generally I use the `/* */` form as docstrings explaining the purpose of the method/function.

Reply:
Oh, usually I see `/** … */` in the role of docstrings; do you prefer `//` for plain comments then?

Reply:
This is now fixed in 9a12afc70230a45afed180756ff1c8d24a0edb37.